app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessToilets.kt

```diff
@@ -28,6 +29,9 @@ class AddWheelchairAccessToilets : OsmFilterQuestType<WheelchairAccessToiletsAns
     override fun createForm() = AddWheelchairAccessToiletsForm()
 
     override fun applyAnswerTo(answer: WheelchairAccessToiletsAnswer, tags: Tags, geometry: ElementGeometry, timestampEdited: Long) {
-        answer.applyTo(tags)
+        when (answer) {
+            is WheelchairAccessToilets -> tags.updateWithCheckDate("wheelchair", answer.access.osmValue)
+            is NoToilet -> throw IllegalStateException()
```

Comment:
Hm on second thought, if `NoToilet` is not a valid choice, but only `WheelchairAccessToilets` is, then this should also be the generic type of this class.